lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -4857,6 +4857,8 @@ Galaxy, including:
   <xs:complexType name="Actions">
     <xs:annotation>
       <xs:documentation xml:lang="en"><![CDATA[
+The functionality of the ``actions`` tag should be replaced by 
+[tool provided metadata](https://planemo.readthedocs.io/en/latest/writing_advanced.html#tool-provided-metadata).
```

Comment:
I just don't know that I think this is true. The XML may be undocumented but at least it is structured and declarative. In terms of how the workflow editor works, job collection, etc.. it seems we can be a lot smarter about output actions. It feels like tool provided metadata should be the fallback (despite me knowing a lot more about it and such).

Reply:
Are these the actions this refers to ? https://docs.galaxyproject.org/en/master/dev/schema.html#tool-outputs-data-actions

Reply:
> Are these the actions this refers to ?

Yes

> I just don't know that I think this is true.

That is such a wonderful sentence ;).

So let's just remove my sentence about the tool provided metadata.